app/src/main/java/tech/relaycorp/letro/awala/message/MessageRecipient.kt

```diff
@@ -0,0 +1,12 @@
+package tech.relaycorp.letro.awala.message
+
+sealed interface MessageRecipient {
+
+    data class User(
+        val veraId: String
```

Comment:
An Awala endpoint id (e.g., `022a3c85609d4d626bc01cd87df71d01f6bb9a62efce214d37b0d4faf4f3ebb74`) and a VeraId member id (e.g., `alice@applepie.rocks`) are two very different things, and the Awala SDK only deals with Awala endpoint ids.

Reply:
Okay, I just added it for readability (how this class will be used), I'll change it later